diffblue.yml

```diff
@@ -1,4 +1,5 @@
 cbmcArguments:
   # Because tic-tac-toe has 9 squares, we need to unwind the loops 10 times
   # This will be auto-detected in a future version
-  max-nondet-array-length: 10
```

Comment:
What does this do to the original tictactoe implementation? which I believe is still in the repository

Reply:
Because now the board is 2d array (3x3), `unwind` needs to be 4(3+1) and I was also told `max-nondet-array-length` cannot be higher than `unwind`. I have updated the comment in the `diffblue.yml`